src/js/common/index.js

```diff
@@ -5,9 +5,6 @@
 // other such libraries in here. Most of the site does not use these legacy
 // frameworks and it belongs in a lower-level module.
 
-// Global site style.
-require('../../sass/style.scss');
```

Comment:
@akainic I'm the exact wrong person to ask this question. But that sounds like a good fix to me. Moving it out of `index.js` definitely solved the problem of double imports. Moving it to `no-react-entry.js` is probably the correct fix.